server/routes/__init__.py

```diff
@@ -1,5 +1,7 @@
+from .contact import contact_bp
 from .user import user_bp
 
-
 def register_routes(app):
     app.register_blueprint(user_bp, url_prefix="/api")
+    app.register_blueprint(contact_bp, url_prefix="/api")
```

Comment:
I think for reusability purposes, refactor the `url_prefix` value to use a top-level declared variable. In case we need to easily switch our PREFIX for all routes, it will be a one-line change.
```suggestion
API = "/api"
def register_routes(app):
    app.register_blueprint(user_bp, url_prefix=API)
    app.register_blueprint(contact_bp, url_prefix=API)
```